.github/workflows/create_release.yml

```
@@ -11,21 +11,26 @@ on:
 
 jobs:
   create_release:
-    runs-on: ubuntu-20.04  # newest available distribution, aka focal
+    runs-on: ubuntu-22.04  # newest available distribution, aka jellyfish
     steps:
       - name: Checkout Repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           # Prevent use of implicit GitHub Actions read-only token GITHUB_TOKEN. We don't deploy on
           # the tag MAJOR.MINOR.PATCH event, but we still need to deploy the maven-release-plugin master commit.
           token: ${{ secrets.GH_TOKEN }}
           fetch-depth: 1  # only need the HEAD commit as license check isn't run
+      - name: Setup java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'  # zulu as it supports a wide version range
+          java-version: '17'  # earliest LTS and last that can compile the 1.6 release profile.
```

```suggestion
          java-version: '17'
```

My comment on the other PR was probably confusing, AFAIU, this comment isn't true on both counts for java 17?

---

sorry this was supposed to be `  # earliest LTS supported by Spring Boot 3` I'll fix the others!